### ISSUE TITLE
🐞 iPhone/Apple Bug - Slider/Carousel arrows appear on top level

### ISSUE THREAD

**Issue by perks98** [elementor/elementor#13791]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

This has been an issue for a very long time. I use the Astra theme which has a flyout navigation menu on mobile plus I have an Elementor popup running on a site, any carousel or slider with arrows on an Apple device, in my testing iPhone, the arrows appear on the top level while on an android phone it acts normal. Can you look into this?

![Capture154](https://user-images.githubusercontent.com/55829113/106762659-f383e180-662d-11eb-8bc6-79506e248e40.PNG)
![Capture155](https://user-images.githubusercontent.com/55829113/106762672-f67ed200-662d-11eb-9e00-e0b315b61e28.PNG)

## Steps to reproduce
Create an Elementor site and create a large popup that appears on mobile. On any page add a widget like a sider or a carousel with arrows. Either with an iPhone or using (https://appetize.io/ <- 60s demo phone emulator) you should see that the arrows go over the popup while it should be under. 

## Isolating the problem

- [x] I can reproduce this bug consistently using the steps above.

**Comment by shilo-ey** [comment]
Thanks for sharing.

This is a known issue with Safari and caused by the fixed element (sidebar menu).

For now, you can add to the specific element the following CSS property: `-webkit-transform: translate3d(0,0,0);`

Thanks

**Comment by perks98** [comment]
Thanks for the reply @shilo-ey.

I'm not too sure where to apply the CSS property you provided as I've tried the slider, arrow buttons and the fly-out menu and it doesn't seem to fix it.

Just to clarify the issue is present on both Safari and Google Chrome browsers of iPhone. Plus as seen in the GIF below (appointment popup) the arrows overlay an Elementor popup as well as the theme mobile menu.

![TT GIF 04-02-2021 08-51-29](https://user-images.githubusercontent.com/55829113/106868594-dcdd9900-66c6-11eb-9ab7-998601a406ad.gif)

This issue has been present for a very long while (before version 3) but I only just got time to point it out.

**Comment by black-eye** [comment]
> @shilo-ey 
> This is a known issue with Safari and caused by the fixed element (sidebar menu).
> 
> For now, you can add to the specific element the following CSS property: `-webkit-transform: translate3d(0,0,0);`


The issue is caused because of using the translate3d in Safari. If normal (2D) translate is used, everything works fine.

In my theme (or custom CSS) I use this code to fix the issue:
```
.elementor-swiper-button {
    // this is the Elementor's default setting:  transform: translate3d(0,-50%, 1px); 
    transform: translate(0,-50%);
}
```
It's way better than reset the translate completely, otherwice the arrows would be off-centered.

 @shilo-ey, please consider using `transform: translate` instead of `transform: translate3d` by default. In fact, there's no reason to use it here (maybe apart from a very little preformance gain).

------
Just a side note: for my CSS I use SCSS with autoprefix applied, so maybe there should be some browser specific prefixes applied there.

**Comment by perks98** [comment]
@black-eye thanks for that, your solution seemed to have worked. Unfortunately, I work for a company that has 150+ sites using Elementor and can't apply this patch to all of them. @shilo-ey do you have a roadmap of when this will be fixed officially?

**Comment by d19dotca** [comment]
Looks like this was fixed in the latest Elementor 3.1.4 version released today.

**Comment by perks98** [comment]
Yeah, can confirm from updating a few sites that the update worked.
